test/hardhat/MetaMorpho.spec.ts

```diff
@@ -250,19 +250,35 @@ describe("MetaMorpho", () => {
   });
 
   it("should simulate gas cost [main]", async () => {
-    for (let i = 0; i < suppliers.length; ++i) {
-      logProgress("main", i, suppliers.length);
+    const nbSuppliers = suppliers.length;
+    const nbDeposits = nbSuppliers * 2;
 
-      const supplier = suppliers[i];
-      const assets = BigInt.WAD * toBigInt(1 + Math.floor(random() * 100));
+    for (let i = 0; i < nbDeposits; ++i) {
+      logProgress("main", i, nbDeposits);
+
+      const j = i >= nbSuppliers ? nbDeposits - i - 1 : i;
```

Comment:
Not exactly the same order, but more sexy 
```suggestion
      const j = i % nbSuppliers;
```